a3p-integration/proposals/p:upgrade-19/agoricNames.test.js

```diff
@@ -135,59 +163,28 @@ test.serial('check all existing values are preserved', async t => {
   );
 });
 
-test.serial('check we can add new chains', async t => {
```

Comment:
Why is it okay to drop this test?

Reply:
> https://github.com/Agoric/agoric-sdk/commit/2195ace1fe125e2c0b9b05c32e1cf147a2604d60 removes  write-chain-info.js from f:fast-usdc. So we simulate what is does (relative to agoricNames) in order to test ephemeral onUpdate callbacks keep working after an agoricNames upgrade.

Because of this.